README.md

```diff
@@ -22,17 +46,15 @@ kustomize build config/examples/mock | kubectl create -f -
 
 ## Manual Testing
 
-## Start NNF Driver
+Start NNF Driver:
```

Comment:
We still need to rip out all these NNF references in the main README, planning on doing that as a part of another PR though.